code/game/gamemodes/cult/talisman.dm

```diff
@@ -148,8 +148,11 @@
 
 	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked
 	var/obj/effect/rune/teleport/actual_selected_rune = potential_runes[input_rune_key] //what rune does that key correspond to?
-	if(!actual_selected_rune)
+	if(!src || qdeleted(src) || !user || user.l_hand != src && user.r_hand != src || user.incapacitated() || !actual_selected_rune)
+		to_chat(user, "<span class='warning'>You must have the talisman in a active hand to operate it!</span>")

```

Comment:
just noticed. If user is `null` and this fails because of that, this will runtime. Just get rid of this line and the `log_game`